### PR TITLE
Added w3cid for autopublish

### PR DESCRIPTION
**Pull request by ianbjacobs**

### CODE REVIEW

**Comment by ianbjacobs** [comment]
@rsolomakhin, @adrianhopebailie I have added bits to help with auto-publishing.

File: index.html [hunk 1]
```diff
@@ -18,27 +18,32 @@
                 {   name:       "Adrian Hope-Bailie",
                     url:        "https://github.com/adrianhopebailie",
                     company:    "Ripple",
-                    companyURL: "https://ripple.com"
-                },
+                    companyURL: "https://ripple.com",
+  		    w3cid: 	42590
```
Comment (rsolomakhin) [inline index.html]: Looks like you need +1 space to align w3cid and companyURL. If this is caused by using tabs vs. spaces, please make sure to follow the file's existing convention.

**Review by rsolomakhin — CHANGES_REQUESTED** [review]

**Comment by ianbjacobs** [comment]
Thanks @rsolomakhin. I ran tidy which should help.

**Review by rsolomakhin — APPROVED** [review]
Thank you for the cleanup!

**Comment by ianbjacobs** [comment]
Cool..if all good, please give green light for merge. Thanks!

**Pull request merged**